droopler.install

```diff
@@ -370,14 +371,26 @@ function droopler_update_8103() {
   } catch (MissingDependencyException $e) {
     $values['droopler__droopler_update_8103_0'] = FALSE;
   }
-
-  // Add new fields to paragraphs.
-  $configs = [
+  // Install Droopler modules.
+  //Make sure that d_p and it's configuration are enabled and imported.
+  $module_installer->install(['d_p'], TRUE);
```

Comment:
There is no way this module is uninstalled, each paragraph has a dependency on it.